File: FitnessWeb.API/Commands/CreateFitnessTipCommand.cs
using MediatR;

namespace FitnessWeb.API.Commands
{
    public class CreateFitnessTipCommand : IRequest<int>
    {
        public int FitnessProgramId { get; set; }
        public string Name  { get; set; }
        public string Description { get; set; }
    }
}

File: FitnessWeb.Frontend/AngularApp/src/commands/createFitnessTipCommand.ts
export class CreateFitnessTipCommand{
    name: string;
    description: string;
    fitnessProgramId: number;
    
    constructor (name: string, description: string, fitnessProgramId: number){
        this.name = name;
        this.description = description;
        this.fitnessProgramId = fitnessProgramId;

    }
}